openeo/api/logs.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+class LogEntry:
+    """
+    A log entry.
+    """
+
+    def __init__(self, entry):
+        self.log_id = entry['id']
+        self.code = entry.get("code", 0)
+        self.level = entry['level']
+        self.message = entry['message']
+        self.time =  entry.get("time", None) # todo: native date/time object?
+        self.path = entry.get("path", [])
+        self.links = entry.get("links", [])
+        if 'data' in entry:
+            self.data = entry['data']
+
+    def toJson(self):
+        return json.dumps(self, default = lambda o: o.__dict__) # doesn't work
```

Comment:
does this work?

     json.dumps(self.__dict__)?

Reply:
Had to add the default in render_component, not here. Removed it completely.